infrastructure/general/setup-bootnode.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Set up a Boot Node
+description: Instructions on setting up, testing, maintaining, and testing a boot node.
+---
+
+!!!note
+    When you first start a node, it has to find a way to find other nodes in the network. For that
+    purpose, you need "bootnodes". After the first bootnode is found, it can use that node’s connections
+    to continue expanding and play its role in the network, like participating as a validator.
+
+<!-- TODO: link wss guide wherever applicable. -->
+
+## Accessing the Bootnode
+
+The consensus is that bootnodes have to be accessible in three ways:
+
+- **P2P**: the p2p port, which can be set by `--listen-addr /ip4/0.0.0.0/tcp/<port>`. This port is
+  not automatically set on a non-validator node (for example, an archive RPC node)
+- **P2P/WS**: the websocket version, which can be set by `--listen-addr /ip4/0.0.0.0/tcp/<port>/ws`
+- **P2P/WSS**: the _secure_ websocket version. An SSL-secured connection to the p2p/ws port must be
+  achieved by a proxy since the node cannot include certificates. It is needed for light clients.
```

Comment:
```suggestion
- **P2P** - the p2p port, which can be set by `--listen-addr /ip4/0.0.0.0/tcp/<port>`. This port is
  not automatically set on a non-validator node (for example, an archive RPC node)
- **P2P/WS** - the websocket version, which can be set by `--listen-addr /ip4/0.0.0.0/tcp/<port>/ws`
- **P2P/WSS** - the _secure_ websocket version. An SSL-secured connection to the p2p/ws port must be
  achieved by a proxy since the node cannot include certificates. It is needed for light clients
```